src/qt/dbb_gui.cpp

```diff
@@ -1320,7 +1320,17 @@ void DBBDaemonGui::eraseAllBackups()
 
 void DBBDaemonGui::verifyBackup(const QString& backupFilename)
 {
-    std::string command = "{\"backup\" : { \"decrypt\": \"yes\", \"check\" : \"" + backupFilename.toStdString() + "\" } }";
+    bool ok;
+    QString tempBackupPassword = QInputDialog::getText(this, "", tr("Enter backup-file password"), QLineEdit::Password, "", &ok);
+    if (!ok || tempBackupPassword.isEmpty())
+        return;
+
+    std::string hashHex = DBB::getStretchedBackupHexKey(tempBackupPassword.toStdString());
+    std::string command = "{\"backup\" : {"
+                "\"check\" :\"" + backupFilename.toStdString() + "\","
+                "\"decrypt\": \"yes\","
```

Comment:
`"\"decrypt\": \"yes\","` can be dropped since the new software will no longer be backward compatible.
(Also for the `seed` and backup creation commands.)


'hidden' api page: https://digitalbitbox.com/api_v2.0.0